fdbserver/SkipList.cpp

```diff
@@ -757,17 +760,33 @@ class SkipList : NonCopyable
 		Version version;
 		bool *result;
 		int state;
+		Standalone<VectorRef<KeyRangeRef>>* conflictingKeyRange; // null if report_conflicting_keys is not enabled.
```

Comment:
Optional would probably be better here.
```suggestion
		Optional<Standalone<VectorRef<KeyRangeRef>>> conflictingKeyRange; // null if report_conflicting_keys is not enabled.
```